src/config/functions.rs

```diff
@@ -28,6 +28,9 @@ impl Parse for Parameter {
                 return None;
             }
         };
+        toml.check_unwanted(&["const", "nullable", "length_of", "name"],
```

Comment:
"pattern" also needed for this and other with `Ident::parse`.
Strange that it don't failed on gtk.